chemberta/masked-lm/train_roberta_mlm.py

```diff
@@ -53,10 +56,13 @@
 flags.DEFINE_float(name="mlm_probability", default=0.15, lower_bound=0.0, upper_bound=1.0, help="")
 
 # Train params
+flags.DEFINE_float(name="frac_train", default=0.95, help="")
+flags.DEFINE_integer(name="eval_steps", default=1000, help="")
+flags.DEFINE_integer(name="logging_steps", default=100, help="")
 flags.DEFINE_boolean(name="overwrite_output_dir", default=True, help="")
-flags.DEFINE_integer(name="num_train_epochs", default=10, help="")
+flags.DEFINE_integer(name="num_train_epochs", default=1, help="")
 flags.DEFINE_integer(name="per_device_train_batch_size", default=64, help="")
-flags.DEFINE_integer(name="save_steps", default=10_000, help="")
+flags.DEFINE_integer(name="save_steps", default=10000, help="")
```

Comment:
Stylistic thing that doesn't make a big diff but I actually like the `_` formatting version as it makes it easy to understand the number without zooming in and counting zeros